internal/gitstream/sync.go

```diff
@@ -102,6 +102,10 @@ func (s *Sync) Run(ctx context.Context) error {
 			return nil
 		}
 
+		if c.Author.Name == "dependabot[bot]" {
```

Comment:
Instead of hardcoding that, I would add a list of usernames for which commits should be ignored in the config. We can make that list a set, and continue (like you did) if `c.Author.Name` is found in that set.

Reply:
PTAL.